Podcast/ActionSheetTableViewCells.swift

```diff
@@ -103,6 +103,94 @@ class ActionSheetRecastDescriptionTableViewCell: ActionSheetStandardTableViewCel
     }
 }
 
+// MARK
+// MARK -- ActionSheetCreateRecastBlurbTableViewCell
+// MARK
+
+protocol ActionSheetCreateRecastBlurbTableViewCellDelegate: class {
+    func didPressSaveBlurb(for cell: ActionSheetCreateRecastBlurbTableViewCell, with blurb: String)
+}
+
+class ActionSheetCreateRecastBlurbTableViewCell: UITableViewCell, ActionSheetTableViewCellProtocol {
+    static let identifier: String = "actionSheetCreateRecastBlurbTableViewCellIdentifier"
+    static var cellHeight: CGFloat = 200
+
+
+    let padding: CGFloat = 18
+    let supplierViewHeight: CGFloat = UserSeriesSupplierView.height
+    let smallPadding: CGFloat = 6
+
+    var textView: UITextView!
+    var supplierView: UserSeriesSupplierView!
+    var postButton: UIButton!
+
+    weak var delegate: ActionSheetCreateRecastBlurbTableViewCellDelegate?
+
+    override init(style: UITableViewCellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        backgroundColor = .offWhite
+        separatorInset = UIEdgeInsets.zero
+
+        selectionStyle = .none
+        supplierView = UserSeriesSupplierView()
+        addSubview(supplierView)
+
+        textView = UITextView()
+        textView.font = ._14RegularFont()
+        textView.textColor = .slateGrey
+        textView.isUserInteractionEnabled = true
+        textView.backgroundColor = .offWhite
+        textView.placeholder = "Add a blurb to your recast..."
+        addSubview(textView)
+
+        postButton = UIButton()
+        postButton.setTitle("Post", for: .normal)
+        postButton.setTitleColor(.sea, for: .normal)
+        postButton.addTarget(self, action: #selector(postButtonPress), for: .touchUpInside)
+        addSubview(postButton)
+
+        supplierView.snp.makeConstraints { make in
+            make.leading.top.trailing.equalToSuperview()
+            make.height.equalTo(supplierViewHeight)
+        }
+
+        textView.snp.makeConstraints { make in
+            make.top.equalTo(supplierView.snp.bottom).offset(padding)
+            make.leading.trailing.equalToSuperview().inset(padding)
+        }
+
+        postButton.snp.makeConstraints { make in
+            make.top.equalTo(textView.snp.bottom).offset(smallPadding)
+            make.trailing.equalToSuperview().inset(padding)
+            make.bottom.equalToSuperview().inset(smallPadding)
+        }
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func setup(withOption option: ActionSheetOptionType) {
```

Comment:
setup(with option:). No need to duplicate the option. setup(for option:) wouldn't be bad either.